projects/cds/src/cms-components/merchandising/carousel/merchandising-carousel/merchandising-carousel.component.spec.ts

```diff
@@ -0,0 +1,177 @@
+import {
+  Component,
+  Input,
+  Pipe,
+  PipeTransform,
+  TemplateRef,
+} from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { RouterTestingModule } from '@angular/router/testing';
+import { Product } from '@spartacus/core';
+import { CmsComponentData } from '@spartacus/storefront';
+import { CmsMerchandisingCarouselComponent } from 'projects/cds/src/cds-models/cms.model';
```

Comment:
please use relative path like the others

Reply:
Good catch, done

Reply:
Wasn't this caught by the production CI build? Just checking myself, because if it wasn't then I missed something.

Reply:
I don't think so as the CI build was green